src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -203,6 +202,13 @@ static string GetLanguageSpecificId(string? language, string noLanguageId, strin
                     message = string.Format(FeaturesResources.The_assembly_0_does_not_contain_any_analyzers, fullPath);
                     break;
 
+                case AnalyzerLoadFailureEventArgs.FailureErrorCode.ReferencesFramework:
+                    // TODO: what are the values for C# and VB?
```

Comment:
@chsienki -- what's the error code for the C# one? It looks like no code was ever added to the command line compiler as far as I can tell?